clio-server/src/test/scala/org/broadinstitute/clio/server/dataaccess/PersistenceDAOSpec.scala

```diff
@@ -41,7 +41,7 @@ class PersistenceDAOSpec extends TestKitSuite("PersistenceDAOSpec") {
     val document2 = DocumentMock.default.copy(mockFieldDouble = Some(0.9876))
 
     for {
-      _ <- dao.initialize(index)
+      _ <- dao.initialize(Seq(index))
```

Comment:
Given the number of places you needed to do this, it would've been less work to add an override that accepted a non-Seq argument.

If we can combine all the tests into a single one the loops over all the indexes that would also remove the need to pass a single index in.